pipeline-runner/tests/testthat/_snaps/gem2s-7-upload_to_aws.md

```diff
@@ -19,12 +19,12 @@
         .. .. ..$ :List of 4
         .. .. .. ..$ key    : chr "123abc"
         .. .. .. ..$ name   : chr "WT1"
-        .. .. .. ..$ color  : chr "#e377c2"
+        .. .. .. ..$ color  : chr "#77aadd"
```

Comment:
I beleive these are changes that are related to when @gerbeldo updated the colors.